core/keyboard_nav/block_navigation_policy.ts

```diff
@@ -171,3 +104,88 @@ export class BlockNavigationPolicy implements INavigationPolicy<BlockSvg> {
     return current instanceof BlockSvg;
   }
 }
+
+/**
+ * Returns a list of the navigable children of the given block.
+ *
+ * @param block The block to retrieve the navigable children of.
+ * @returns A list of navigable/focusable children of the given block.
+ */
+function getBlockNavigationCandidates(block: BlockSvg): IFocusableNode[] {
+  const candidates: IFocusableNode[] = block.getIcons();
+
+  for (const input of block.inputList) {
+    if (!input.isVisible()) continue;
+    candidates.push(...input.fieldRow);
```

Comment:
Does this preserve the behaviour that @RoboErikG has been working on with the clickable/nonclickable image fields?

Reply:
@RoboErikG or @gonfunko This is okay to merge once this is confirmed.

Reply:
Confirmed this is the same logic and it works with my test block.